code/modules/jobs/access.dm

```diff
@@ -6,6 +6,8 @@
 	//check if it doesn't require any access at all
 	if(check_access(null))
 		return TRUE
+	if(!accessor) //likely a TK user.
```

Comment:
this seems like more hiding the problem than fixing it, I'm not really a fan.
It's also probably affecting more than just doors...

Reply:
#63116 removed the sanity check that was in place specifically to prevent a runtime, causing the regression since you can't send signals to null things. `allowed(null)` is by design. 

Nothing below the line would change in behavior from having it added aside since EVERYTHING is already using istype checks, and ALWAYS has as far back as the repo history will go.
![image](https://user-images.githubusercontent.com/6209658/183816836-ddb94903-889a-4081-bcc7-ce9fef750757.png)